plugin/src/main/kotlin/tanvd/grazi/utils/JLangUtils.kt

```diff
@@ -46,3 +50,17 @@ fun FlowOrPhrasingContent.toCorrectHtml(example: IncorrectExample) {
         }
     }
 }
+
+object LangToolInstrumentation {
+    fun registerLanguage(lang: Lang) {
```

Comment:
I'm not like that registerLanguage is available for all. You can call this function only once after download jar, in other cases it will be NPE or undefined behaviour. Maybe mark it as internal or something?